frontend/components/groups/js/groups.component.ts

```diff
@@ -258,7 +258,7 @@ class GroupsController implements ng.IController {
 		const escapable = true;
 		this.dialogService.confirm(`Confirm Delete`, content, escapable, "Yes", "Cancel")
 			.then(() => {
-				this.groupsService.deleteAllGroups(this.teamspace, this.model);
+					this.groupsService.deleteGroups(this.teamspace, this.model, this.groupsToShow);
```

Comment:
Maybe Line 349 of deleteGroups in groups.service needs a check to make sure groupsToShow is not null?